members/hanSiang.md

```diff
@@ -0,0 +1,14 @@
+# Han Siang PHUA
+
+## Education
+
+## Work Experience
+
+* [3 months: 2017 Jun - Aug] Sofware Engineering Intern, at MyMiniFactory, London, UK
+* [5 months: 2016 May - Sep] Mechanical Engineering Consultant Intern, at Surbana Jurong, Singapore
+* 
+
+## Accomplishments
+
+1. Repurposed and rebranded an existing web application for a different market: During my internship at MyMiniFactory, I repurposed a 40 KLoC software to suite a different market. The new webapp is now live.
```

Comment:
Might want to use bullets instead of numbering to maintain consistency within the document.